QuickOSM/core/actions.py

```diff
@@ -16,17 +23,41 @@
 ACTIONS_VISIBILITY = ['Canvas', 'Feature', 'Field']
 
 
-def add_actions(layer: QgsVectorLayer, keys: list):
+def add_actions(layer: QgsVectorLayer, keys: list, visibility: str = None, dialog: QDialog = None):
```

Comment:
You can access `iface` with `from qgis.utils import iface`. No need for the dialog.

Reply:
I saw QgsAction is using string base for the scope. You can make an enum for it ? So by just reading the annotation, e know what to give and the range of possible values.

Reply:
I need the dialog to access the reload_action to add the action for the layer